src/features/movie/components/overview/Overview.tsx

```diff
@@ -0,0 +1,26 @@
+import { Movie } from "models/movie.model";
+
+import styled from "./overview.module.scss";
+
+export const Overview: React.FunctionComponent<Movie> = (movie) => {
+  const { description, actors } = movie;
+
+  return (
+    <div className={styled.overview}>
+      <h3>Overview</h3>
+      <div role="document" className={styled.description}>
+        {description}
+      </div>
+      <div className={styled.actors}>
+        {actors.map((actor) => {
+          return (
```

Comment:
You can omit curly braces and return statement for shorthand syntax.

Reply:
Done